install/ansible/tasks/bootstrap_centos.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name: "Install required packages on CentOS "
+  yum:
+    name: "{{ item }}"
+    state: latest
+    with_items:
+      - unzip
+      - tar
+      - btrfs-progs-devel
+      - device-mapper-devel
```

Comment:
Why do we need device mapper?

Reply:
Somehow was under the impression that device mapper was needed for centos.